### ISSUE TITLE
Plugin roadmap

### ISSUE THREAD

**Issue by fvisticot** [rodydavis/easy_google_maps#5]
Great plugin !!
I think lot of people are waiting for this plugin.
Do you plan to add new features ?  :
- markers
- polygon
- moveCamera
- polyline

**Comment by rodydavis** [comment]
Not really. This is meant for a 1 marker and address or query setup. Maybe in the future I can add different widgets in the package but the EasyGoogleMaps will aim to have the simplest api and minimal set up!

I’m open to PRs though! I use google maps in another project of mine with 65,000 markers but for that I have to use SQLite and it can get much more complicated for the dev.

**Comment by fvisticot** [comment]
Did you have a tutorial explaining what you have done for this plugin ?
Are you using native JS library ?
With some information some people or myself will try to play with google map for web.

I think this kind of plugin is very usefull for many web applications ?

**Comment by rodydavis** [comment]
Right now on web I am using the iframe and url scheme approach with some clever flutter widget sizing!
